src/Core/LegendTexts.js

```diff
@@ -1,143 +1,120 @@
             },
             QuestionsDetails: {
-                title: <Trans>Fragen und Antworten</Trans>,
-                text: <Trans i18nKey="exams.mc.questions.text">
-                    <p>Hier kannst du dir alle Fragen der Semesterprüfung mit den dazugehörigen Antworten ansehen. Jede
+                title: 'Fragen und Antworten',
+                text: <p>Hier kannst du dir alle Fragen der Semesterprüfung mit den dazugehörigen Antworten ansehen. Jede
                         Frage wurde mit verschiedenen Tags versehen, mit deren Hilfe du die Fragen filtern kannst. Die
                         Filter schwer und leicht zeigen dir wie viel deiner Mitstudierenden diese Frage beantworten
                         konnten. Schwere Fragen sind Fragen, die weniger als 40% deiner Mitstudierenden beantworten
                         konnten. Leichte Fragen hingegen konnten mehr als 80% deiner Kommilitonen beantworten.So kannst
                         du dir z.B. nur die Fragen anzeigen lassen, die du falsch beantwortest hast.</p>
-                </Trans>
+                
             }
         },
         Ptm: {
             Totals: {
-                title: <Trans>Gesamtes Ergebnis</Trans>,
-                text: <Trans>Hier findest du eine generelle Auswertung deines PTMs im Vergleich mit der Kohorte.</Trans>
+                title: 'Gesamtes Ergebnis',
+                text: 'Hier findest du eine generelle Auswertung deines PTMs im Vergleich mit der Kohorte.'
             },
             Timeline: {
-                title: <Trans>Deine Entwicklung</Trans>,
-                text: <Trans>Hier findest du eine generelle Auswertung deines PTMs im zeitlichen Verlauf.</Trans>
+                title: 'Deine Entwicklung',
+                text: 'Hier findest du eine generelle Auswertung deines PTMs im zeitlichen Verlauf.'
             },
             Strengths: {
-                title: <Trans>Starke Fächer in diesem PTM</Trans>,
-                text: <Trans>Hier findest du eine Übersicht zu deinen besten theoretischen, klinischen und
-                    Querschnittsfächern im PTM diesen Semesters. Die Auswertung beruht auf dem Verhältnis der richtig
-                    beantworteten im Vergleich zu allen beantworteten Fragen.</Trans>,
+                title: 'Starke Fächer in diesem PTM',
+                text: 'Hier findest du eine Übersicht zu deinen besten theoretischen, klinischen und Querschnittsfächern im PTM diesen Semesters. Die Auswertung beruht auf dem Verhältnis der richtig beantworteten im Vergleich zu allen beantworteten Fragen.',
             },
             Subjects: {
-                title: <Trans>Alle Fächer</Trans>,
-                text: <Trans>
-                    <p>Hier findest du eine Übersicht zu deinen Leistungen in den theoretischen, klinischen und
+                title: 'Alle Fächer',
+                text: <Fragment><p>Hier findest du eine Übersicht zu deinen Leistungen in den theoretischen, klinischen und
```

Comment:
<Fragment> und <p> tags sollten in Zukunft nicht in dieser Datei drinstehen. Diese Aufgabe gehört aber nicht zu diesem Ticket (Trans raus, #2514) - daher ist der Code hier ok. ✔️
Ich mache ein neues Ticket extra für die Bereinigung von LegendTexts.js .

Reply:
alles  klar, Danke